requirements.txt

```diff
@@ -1,16 +1,16 @@
-Flask>=1.1.2
-Jinja2>=2.11.2
-werkzeug>=1.0.1
-Flask-DebugToolbar>=0.11.0
+Flask>=2.1.0
+Jinja2>=3.0
+itsdangerous >= 2.0
+click >= 8.0
+Werkzeug>=2.0
+Flask-DebugToolbar>=0.13.1
 Flask-UUID>=0.2
-sentry-sdk[flask]>=0.20.2
+sentry-sdk[flask]>=1.5.8
 certifi
-redis>=3.5,<4.0
+redis>=4.2.2
 msgpack-python==0.5.6
-requests>=2.23.0
-SQLAlchemy>=1.3.16
+requests>=2.27.1
+SQLAlchemy>=1.3.16,<2.0
 mbdata==25.0.4
 sqlalchemy-dst>=1.0.1
-importlib-metadata>=3.10.0
-itsdangerous==2.0.1
-MarkupSafe==2.0.1
\ No newline at end of file
+importlib-metadata>=3.10.0; python_version < '3.10'
```

Comment:
this might have broken our magic stuff that loads requirements.txt into setup.py install_requires